R/tm_t_binary_outcome.R

```diff
@@ -438,6 +447,17 @@ tm_t_binary_outcome <- function(label,
                                 default_responses =
                                   c("CR", "PR", "Y", "Complete Response (CR)", "Partial Response (PR)", "M"),
                                 rsp_table = FALSE,
+                                control = list(
+                                  global = list(
+                                    method = ifelse(rsp_table, "clopper-pearson", "waldcc"),
+                                    conf_level = 0.95
+                                  ),
+                                  unstrat = list(
+                                    method_ci = ifelse(rsp_table, "wald", "waldcc"),
+                                    method_test = "schouten", odds = TRUE
+                                  ),
```

Comment:
it's easy to miss that this list should have three elements

```suggestion
                                  unstrat = list(
                                    method_ci = ifelse(rsp_table, "wald", "waldcc"),
                                    method_test = "schouten", 
                                    odds = TRUE
                                  ),
```

Reply:
Updated.